src/backend/test/unit/pia-intake/pia-intake.service.spec.ts

```diff
@@ -136,6 +136,78 @@ describe('PiaIntakeService', () => {
         'updatedByDisplayName',
       ]);
 
+      expect(result).toEqual(getPiaIntakeRO);
+    });
+    it('succeeds calling the database and update submittedAt column when the drafter submit pia for', async () => {
+      const createPiaIntakeDto: CreatePiaIntakeDto = {
+        ...createPiaIntakeMock,
+        submittedAt: null,
+      };
+      const piaIntakeEntity = { ...piaIntakeEntityMock };
+      const getPiaIntakeRO = { ...getPiaIntakeROMock };
+
+      const user: KeycloakUser = { ...keycloakUserMock };
+
+      piaIntakeRepository.save = jest.fn(async () => {
+        delay(10);
+        return piaIntakeEntity;
+      });
+
+      omitBaseKeysSpy.mockReturnValue(getPiaIntakeRO);
+
+      const result = await service.create(createPiaIntakeDto, user);
+
+      expect(piaIntakeRepository.save).toHaveBeenCalledWith({
+        ...createPiaIntakeDto,
+        createdByGuid: user.idir_user_guid,
+        createdByUsername: user.idir_username,
+        updatedByGuid: user.idir_user_guid,
+        updatedByUsername: user.idir_username,
+        updatedByDisplayName: user.display_name,
+        drafterEmail: user.email,
+        submittedAt: new Date(),
```

Comment:
The test might have failed here.

You can only check submittedAt is defined and is an instanceof Date.